deployments/helm-chart/values.yaml

```diff
@@ -221,17 +221,22 @@ controller:
   ## The number of replicas of the Ingress Controller deployment.
   replicaCount: 1
 
-  ## A class of the Ingress Controller.
+  # Configures the ingress class the Ingress Controller uses.
+  ingressClass:
+    ## A class of the Ingress Controller.
 
-  ## IngressClass resource with the name equal to the class must be deployed. Otherwise,
-  ## the Ingress Controller will fail to start.
-  ## The Ingress Controller only processes resources that belong to its class - i.e. have the "ingressClassName" field resource equal to the class.
+    ## IngressClass resource with the name equal to the class must be deployed. Otherwise,
+    ## the Ingress Controller will fail to start.
+    ## The Ingress Controller only processes resources that belong to its class - i.e. have the "ingressClassName" field resource equal to the class.
 
-  ## The Ingress Controller processes all the resources that do not have the "ingressClassName" field for all versions of kubernetes.
-  ingressClass: nginx
+    ## The Ingress Controller processes all the resources that do not have the "ingressClassName" field for all versions of kubernetes.
+    name: nginx
 
-  ## New Ingresses without an ingressClassName field specified will be assigned the class specified in `controller.ingressClass`.
-  setAsDefaultIngress: false
+    ## Creates a new IngressClass object with the name "controller.ingressClass.name". Set to "False" to use an exising IngressClass with the same name. If you use helm upgrade, do not change the value from the previous release. If you are upgrading from a release earlier than 3.3.0, do not set the value to false.
+    create: True
```

Comment:
@haywoodsh can we change this to lowercase here and in docs to keep it inline with other booleans?

Reply:
Changed in the latest commit